### PR TITLE
release/1.1.0

### PR DESCRIPTION
**Pull request by sven1103**
Stable release for deployment on production.

### CODE REVIEW

File: .qube.yml
Comment (KochTobi) [inline .qube.yml:17]: Does not fit the previous readme information

File: .qube.yml
Comment (KochTobi) [inline .qube.yml:13]: Is this correct?

File: AUTHORS.rst
Comment (KochTobi) [inline AUTHORS.rst:13]: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners

This should either be CODEOWNERS or be removed.

File: allow_tls.properties
Comment (KochTobi) [inline allow_tls.properties:1]: We still need to keep an eye on this.

File: docs/authors.rst
Comment (KochTobi) [inline docs/authors.rst:1]: Should not be there in my opinion. Authors are long overhauled with code owners or no specific information at all.
https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners

File: docs/conf.py
Comment (KochTobi) [inline docs/conf.py:50]: Again different copyright after the template merge

File: docs/conf.py
Comment (KochTobi) [inline docs/conf.py:51]: Debatable information

File: docs/conf.py
Comment (KochTobi) [inline docs/conf.py:132]: What is done here?

File: docs/index.rst
Comment (KochTobi) [inline docs/index.rst:10]:
```suggestion
  
```

File: src/main/groovy/life.qbic.portal.portlet/QbicPortlet.groovy
Comment (KochTobi) [inline src/main/groovy/life.qbic.portal.portlet/QbicPortlet.groovy:62]: Please remove this pointless class. I think it was added during a template merge.

File: src/main/resources/.gitignore
Comment (KochTobi) [inline src/main/resources/.gitignore:1]: This is probably no longer needed since the `developer.properties` are now included in the main `.gitignore` I think we could remove it or leave it in place as an extra precaution.

**Review by KochTobi — CHANGES_REQUESTED** [review]
Overall LGTM. Please remove the example class. The other comments are regarding the Qube template I think.

File: src/main/groovy/life.qbic.portal.portlet/QbicPortlet.groovy
Comment (sven1103) [reply src/main/groovy/life.qbic.portal.portlet/QbicPortlet.groovy:62]: It is not pointless, but not needed anymore. Please adhere to the review best practise and avoid doubtable language.

File: docs/index.rst
Comment (sven1103) [reply docs/index.rst:10]: As this is part of the QUBE template structure, we should make the change their and create a new template release.

File: .qube.yml
Comment (sven1103) [reply .qube.yml]: Changed

File: .qube.yml
Comment (sven1103) [reply .qube.yml]: Changed

File: AUTHORS.rst
Comment (sven1103) [reply AUTHORS.rst]: This is part of the QUBE template and needs to be adjusted there.

File: docs/conf.py
Comment (sven1103) [reply docs/conf.py]: Changed

**Review by KochTobi — APPROVED** [review]

File: .gitignore
Comment (KochTobi) [reply .gitignore:29]: @sven1103 What about this change?

**Comment by KochTobi** [comment]
The project does not build

**Comment by sven1103** [comment]
[Make a new release of the Image Viewer Portlet, with zenodo DOI](https://trello.com/c/dt8esQOd/313-make-a-new-release-of-the-image-viewer-portlet-with-zenodo-doi)

**Pull request merged**
